packages/remix-forms/src/createField.tsx

```diff
@@ -140,6 +140,18 @@ type SmartInputProps = {
   a11yProps?: Record<`aria-${string}`, string | boolean | undefined>
 }
 
+const FieldContext = React.createContext<
+  Partial<Omit<Field<never>, 'name'>> | undefined
+>(undefined)
+
+export function useField() {
```

Comment:
what do yo think of the name `userFieldContext` ? It seems to make clearer that the hook is returning a context and creates a neat parallel with the `useFormContext` provided by react-hook-form.
@danielweinmann thoughts?

Reply:
I don't like useFormContext and I think the only reason they named it like that is because useForm was already taken 😂